charts/opentelemetry-collector/values.yaml

```diff
@@ -334,6 +334,40 @@ lifecycleHooks: {}
 #       - /test/sleep
 #       - "5"
 
+# liveness probe configuration
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+##
+livenessProbe:
+  # Number of seconds after the container has started before startup, liveness or readiness probes are initiated.
+  initialDelaySeconds: 1
+  # How often in seconds to perform the probe.
+  periodSeconds: 10
+  # Number of seconds after which the probe times out.
+  timeoutSeconds: 1
+  # Minimum consecutive failures for the probe to be considered failed after having succeeded.
+  failureThreshold: 1
+  # Duration in seconds the pod needs to terminate gracefully upon probe failure.
+  terminationGracePeriodSeconds: 10
+  httpGet:
```

Comment:
Since the healthcheckextension allows setting a specific path lets also allow setting a specific path as part of `httpGet` in both probes.